test/src/lib/traces/devtools-timeline-model.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+/* global describe, it, WebInspector */
+
```

Comment:
use `const WebInspector = require('../../../../../../../../../../../../../../../src/lib/web-inspector');` instead of relying on global `WebInspector`